### PR TITLE
get more resources back to Galaxy main

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

File: resources.yaml [hunk 1]
```diff
@@ -80,7 +80,7 @@ deployment:
     image: htcondor-secondary
     secondary_htcondor_cluster: true
   worker-c28m225-htcondor-secondary:
```
Comment (sanjaysrikakulam) [inline resources.yaml:82]: This flavor is used only for the training. We do not use them for our regular queue.

File: resources.yaml [hunk 2]
```diff
@@ -80,7 +80,7 @@ deployment:
...
-    count: 0 #7
+    count: 7
```
Comment (sanjaysrikakulam) [inline resources.yaml:83]: This flavor is used only for the training. We do not use them for our regular queue. So, this should be `0`.

Comment (sanjaysrikakulam) [inline resources.yaml:83]:
```suggestion
    count: 0
```

**Review by sanjaysrikakulam — CHANGES_REQUESTED** [review]

File: resources.yaml [hunk 3]
```diff
@@ -67,7 +67,7 @@ deployment:
       size: 1024
       type: default
   worker-c28m475-htcondor-secondary:
-    count: 8 #19
+    count: 10 #19
```
Comment (bgruening) [inline resources.yaml:70]: Where are the other 9 ?

Comment (sanjaysrikakulam) [reply resources.yaml:70]: The numbers are a bit off. We have adjusted and redistributed the flavors. I have asked Manuel the same question and have not received an update yet. I will check with him again.

**Review by sanjaysrikakulam — APPROVED** [review]

**Comment by sanjaysrikakulam** [comment]
The TIaaS queues of the corresponding trainings are empty. So, merging this now to release the resources.

**Pull request merged**